packages/amplify-e2e-tests/src/__tests__/amplify-configure.test.ts

```diff
@@ -53,6 +73,8 @@ function testAmplifyConfigureValidation(): Promise<void> {
       .wait('You must enter a valid secretAccessKey')
       .sendLine(validMockAWSSecretAccessKey)
       .wait('Profile Name:')
+      // 'config-test' is a new profile name using defaultAWSSecretAccessKey and validMockAWSSecretAccessKey.
+      // Do NOT use TEST_PROFILE_NAME, otherwise, it will override the default profile.
```

Comment:
```suggestion
      // Fake credentials used in this test are always invalid therefore
      // 'config-test' is a new profile name using defaultAWSSecretAccessKey and validMockAWSSecretAccessKey
      // to avoid overwriting profiles used in other tests.
      // Do NOT use TEST_PROFILE_NAME here, otherwise, it will override the default profile.
```